### PR TITLE
feat: add guardian games slot (only to class item)

### PR DESCRIPTION
**Pull request by Mijago**
I basically add a new category "GuardianGamesClassItem" and show it in the "class item" slot.
I can toggle it via an environment flag, so we can easily disable it (or remove it) when we do not need it anymore.

This felt nicer for the end user than adding an extra `advanced setting`.

for #134

### CODE REVIEW

**Comment by abea** [comment]
The option would fit in with the one for Festival of the Lost, fwiw. Or FOL could switch to this pattern.

**Comment by Mijago** [comment]
Thank you for bringing this up. I will move fotl to this pattern.

**Comment by Mijago** [comment]
Adds the Guardian Games slot only to the class item slot:
image
image


NOT visible for other slots:
image

File: src/app/services/bungie-api.service.ts [hunk 1]
```diff
@@ -418,6 +418,10 @@ export class BungieApiService {
 
 
   private getArmorPerk(v: DestinyInventoryItemDefinition): ArmorPerkOrSlot {
+    // Guardian Games
+    if (environment.featureFlags.enableGuardianGamesFeatures && ( v.hash === 1013401891 || v.hash === 366019830 || v.hash == 537041732) )
```
Comment (TheYeshir13) [inline src/app/services/bungie-api.service.ts:422]: image
image
image

**Review by TheYeshir13 — APPROVED** [review]
Looks good :) Ready for merge

Same Logic as before but does not search for the slot instead returns the specific items (for each class)

Can be turned off which is good

**Pull request merged**